lib/src/main/java/io/ably/lib/realtime/ChannelBase.java

```diff
@@ -213,13 +214,13 @@ void reattach(List<QueuedMessage> messagesToTransfer) {
 
             }
         }
-        attach(true,null);
```

Comment:
why remove this?

Reply:
I think when running tests I had a wrong impression that this reattach will happen as a result of another state transition as I was seeing channels reattached. but now I see they are stuck in attaching state. It's a result of writing things in hurry without having time to check their effect. I'll reintroduce this.